frontend/src/components/c-summary-charts.vue

```diff
@@ -258,6 +270,19 @@
           v-if="filterGroupSelection === 'groupByNone' && sortGroupSelection.includes('totalCommits')"
         ) {{ getPercentile(j) }} %&nbsp
           span.tooltip-text.right-aligned {{ getPercentileExplanation(j) }}
+        .summary-chart__title--tags(
+          v-if="(filterGroupSelection === 'groupByAuthors' || filterGroupSelection === 'groupByNone')\
+            && viewRepoTags"
+        )
+          a.tag(
+            v-for="tag in getTags(repo, user)",
+            v-bind:href="getTagLink(repo[0], tag)",
```

Comment:
Is it correct to always get the link from the first repo?

Reply:
should be yes, because this part only runs when we groupByRepos, and the array is of users of the same repo. meaning each link retrieved from the repo is the same!

Reply:
Sorry, might be misunderstanding but isn't this part run when we groupByAuthors?
```
    .summary-chart__title--tags(v-if="isViewingTagsByAuthor")
          a.tag(
            v-for="tag in getTags(repo, user)",
            v-bind:href="getTagLink(repo[0], tag)",
```